src/database/schema_metadata.py

```
@@ -1177,18 +1177,25 @@ def _schema_backed_metadata(
+            {"name": "レコード識別区分", "type": "TEXT", "description": "1:コース、2:G1", "example": "1", "nullable": False},
+            {"name": "開催年", "type": "INTEGER", "description": "レコード樹立レースの開催年", "example": "2026", "nullable": False},
+            {"name": "開催月日", "type": "INTEGER", "description": "レコード樹立レースの開催月日", "example": "816", "nullable": False},
+            {"name": "特別競走番号", "type": "TEXT", "description": "G1レコードの識別番号", "example": "1234", "nullable": False},
             {"name": "競馬場コード", "type": "TEXT", "description": "競馬場コード", "example": "05", "nullable": False},
+            {"name": "開催回", "type": "INTEGER", "description": "開催回", "example": "3", "nullable": False},
+            {"name": "開催日目", "type": "INTEGER", "description": "開催日目", "example": "8", "nullable": False},
+            {"name": "レース番号", "type": "INTEGER", "description": "レース番号", "example": "11", "nullable": False},
+            {"name": "競走種別コード", "type": "TEXT", "description": "競走年齢・障害区分", "example": "13", "nullable": False},
             {"name": "距離", "type": "TEXT", "description": "コース距離", "example": "2400", "nullable": False},
             {"name": "トラックコード", "type": "TEXT", "description": "トラック種別", "example": "10", "nullable": False},
             {"name": "レコードタイム", "type": "TEXT", "description": "レコードタイム（分秒形式）", "example": "2:22.1", "nullable": True},
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)